gitpod-ws.code-workspace

```diff
@@ -0,0 +1,43 @@
+{
+    "folders": [
+        { "path": "./nf-training" }
+    ],
+    "settings": {
+        "typescript.tsdk": "gitpod/node_modules/typescript/lib",
+        "[javascript]": {
+            "editor.defaultFormatter": "esbenp.prettier-vscode",
+            "editor.formatOnSave": true
+        },
+        "[typescript]": {
+            "editor.defaultFormatter": "esbenp.prettier-vscode",
+            "editor.formatOnSave": true
+        },
+        "[json]": {
+            "editor.insertSpaces": true,
+            "editor.tabSize": 2
+        },
+        "[yaml]": {
+            "editor.insertSpaces": true,
+            "editor.tabSize": 2
+        },
+        "[go]": {
+            "editor.formatOnSave": true
+        },
+        "[tf]": {
+            "editor.insertSpaces": true,
+            "editor.tabSize": 2
+        },
+        "go.formatTool": "goimports",
```

Comment:
Not sure that we need the Go-specific language stuff..

Reply:
Good point, that should be removed